module3/projects/take_home/presentation_guidelines.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Take-Home Challenge Presentation Guidelines
+layout: page
+---
+
+## Presentation Guidelines
+
+This guide will serve as a general outline for how students can present their code during a take-home challenge presentation. 
+
+<section class="note">
+### Note
+
+Keep in mind that some companies will have their own guidelines on presenting which takes precenedence over this guideline.
+</section>
+
+**Prior to the presentation:**
+
+- Clear your screen of all windows or tabs except the ones relevant to your presentation. 
+
+- Have any servers fired up and ready to go.
+
+<!-- Note: Do we want them to demo in postman if there is a frontend? -->
+- Have Postman setup in advance to show your endpoints (BE).
+
+
+## Suggested steps for presenting your project:
+### 1. Overview
+  - Start with an overview of the app that you built, summarizing the tasks of the challenge. This can be as simple as verbally stating the main functions of the app.
+
+### 2. Functionality
+
+- **Back End**
+  - Start with Postman and show how your app works through all your endpoints.
+  - If you have a testing suite, now would be a great time to run it and show your test coverage.
+
+- **Front End**
+  - Run through the app, clicking on all the links, entering text into the fields, show the user experience of the application, any responsive design, etc.
+
+
+### 3. Readme & Planning
+
+ - Show your README (because of course you have a thorough, well-documented readme 🙂)
+  Use this as a guide to show how you organized your application (schema, component tree,  endpoints, screenshots or gifs of the UX/UI, etc.)
+
+ - Show your project board and any tools you used in the planning process.
+  
+ - Share any clarifying questions you asked the company and how that guided your process.
+
+  Steps 2 and 3 are great places to talk about any challenges you experienced and how you overcame them, what resources you used, why you choose certain design decisions (page layout, filter options, routes, schema, etc.), and what features or changes you would implement next if you had more time to work on it. 
+
+### 4. Code
+
+  - Highlight 2-3 pieces of code that you are particularly proud of, either because it was a challenge and you figured it out, it’s something new you implemented, or because that piece of code makes you feel like a wizard! No need to walk through the fine-details of how the code is working, you can safely assume your reviewer is familiar with how React or Rails is working. Of course if they ask specific questions, then you can get into the nitty-gritty. 
```

Comment:
replace "nitty-gritty" with other phrasing (etymology of that phrase is problematic)